x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -0,0 +1,277 @@
+[[security-basic-setup]]
+=== Set up basic security for the Elastic Stack
+++++
+<titleabbrev>Set up basic security</titleabbrev>
+++++
+
+Let's say you want to set up the Elastic Stack with some security, but don't
+need to encrypt all traffic flowing in and out of your cluster. After adding
+password protection in the <<security-minimal-setup,minimal security configuration>>, you'll need to configure Transport Layer Security (TLS). The
+transport layer handles all internal communication between nodes and
+communication with nodes in a remote cluster.
+
+The transport layer relies on mutual TLS for both encryption and
+authentication. Applying TLS ensures that a malicious node cannot join the
+cluster and exchange data with other nodes. While implementing username and
+password authentication at the HTTP layer is useful for securing a local
+cluster, there is no internode security without configuring TLS.
+
+Configuring TLS is the basic security setup to prevent unauthorized nodes from
+accessing to your cluster.
+
+[[basic-setup-prerequisites]]
+==== Prerequisites
+
+Complete all steps in <<security-minimal-setup,Minimal security for the Elastic Stack>> on every node in your cluster. After enabling {es} security and setting
+passwords for the built-in users, you can encrypt communications between your
+nodes with TLS.
+
+[[generate-certificates]]
+==== Generate the certificate authority
+
+You can add as many nodes as you want in a cluster but they must be able to communicate with each other. The communication between nodes in a cluster is handled by the transport module. To secure your cluster, you must ensure that the internode communications are encrypted and that mutual TLS is configured.
+
+In a secured cluster, {es} nodes use certificates to identify
+themselves when communicating with other nodes.
+
+The cluster must validate the authenticity of these certificates. The
+recommended approach is to trust a specific certificate authority (CA). When
+nodes are added to your cluster they can use a certificate signed by the same
+CA.
+
+For the transport layer, we recommend using a separate, dedicated CA instead
+of an existing, possibly shared CA. Use the `elasticsearch-certutil` tool to
+generate a CA for your cluster.
+
+NOTE: If you're running a multi-node cluster, jump to
```

Comment:
It's not an "If" here, right? We are describing how to setup security in a multi-node cluster. We are already talking about "nodes" everywhere here